app/controllers/api/public/v1/dossiers_controller.rb

```diff
@@ -11,7 +11,10 @@ def create
     dossier.build_default_individual
     if dossier.save
       dossier.prefill!(PrefillParams.new(dossier, params.to_unsafe_h).to_a)
-      render json: { dossier_url: commencer_url(@procedure.path, prefill_token: dossier.prefill_token) }, status: :created
+      render json: {
+        dossier_url: commencer_url(@procedure.path, prefill_token: dossier.prefill_token),
+        dossier_id: dossier.to_typed_id
```

Comment:
```suggestion
        dossier_id: dossier.to_typed_id,
        dossier_number: dossier.id
```
Désolé pour le pinaillage, je pense que ça serait une bonne idée d'exposer aussi `dossier_number`. C'est utilisé par pas mal d'administration comme référence métier.

Reply:
OK ça marche ! C'est fait :heavy_check_mark: